ruby-gem/spec/api_maker/preloading/has_many_spec.rb

```diff
@@ -33,7 +33,9 @@
 
     expect(preloader.models).to eq [task]
     expect(sql).to include "JOIN tasks AS accessible_table ON accessible_table.id = tasks.id"
-    expect(sql).to include '(EXISTS (SELECT 1 FROM "tasks" WHERE "tasks"."user_id" = 1 AND (tasks.id = accessible_table.id)))'
+    expect(sql).to include 'EXISTS (SELECT 1 FROM "tasks" WHERE ((' \
```

Comment:
Consistent double quotes :-) 
```suggestion
    expect(sql).to include "EXISTS (SELECT 1 FROM \"tasks\" WHERE ((" \
```